scripts/.eslintrc.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = {
+    'env': {
+        'commonjs': true,
+        'node': true,
+        'es6': true
+    },
+    'extends': 'eslint:recommended',
+    'parserOptions': {
+        'ecmaVersion': 12
+    },
+    'rules': {
```

Comment:
Do we need all of these customizations for eslint rules? For tests and code samples we simply use `eslint:recommended`:
https://github.com/hazelcast/hazelcast-nodejs-client/blob/master/test/.eslintrc
https://github.com/hazelcast/hazelcast-nodejs-client/blob/master/code_samples/.eslintrc

Reply:
You can check this page to see which rules are in eslint:recommended. https://eslint.org/docs/rules/
In my first look what I see is, the most important rules such as quotes, semicolons, max-len and no-multiple-empty-lines are not included. No need to mention others.

Reply:
Thanks for the clarification. Then it would be nice to include these changes into configs for tests and code samples. Even better would be if we could somehow share the config to avoid duplication (the only caveat is tests which require mocha plugin).

Reply:
Maybe we could do it with "extends", I will check it out

Reply:
I did this extension mechanism. You can check it out